Docker image broken

[Last commit](https://github.com/compiler-explorer/gcc-cross-builder/commit/1105fbe7c0fa54168f5d63591f1667a0cbf0ce9e) bumped ubuntu version from 18.04 to 20.04.
This breaks old crosstool-ng as some configure script panics on bash that is now `4.x`:

```
configure: error: could not find bash >= 3.1
```
This is stupid as script are simply not ready for anything above 3. A patch like this:
```
+-                     |$EGREP '^GNU bash, version (3\.[1-9]|4)')
++                     |$EGREP '^GNU bash, version (3\.[1-9]|[4-9])')
```

fixes the issue, but we don't really need to be sure we can build all cross compiler on recent distribution.

The GCC11 issue has been fixed by https://github.com/compiler-explorer/gcc-cross-builder/commit/927b99730f8cada3037d2250ac4d49a105c96e3c

(which I broke in the first place in https://github.com/compiler-explorer/gcc-cross-builder/commit/5770b9ed941bd4c28534780f340f64454737c1a2 when I pulled our custom GCC11 to build all the compilers.

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM ubuntu:20.04
+FROM ubuntu:18.04
 MAINTAINER Matt Godbolt <matt@godbolt.org>
 
 ARG DEBIAN_FRONTEND=noninteractive

